src/components/Home.js

```diff
@@ -0,0 +1,5 @@
+const Home = () => (
+  <h2>Home</h2>
+);
+
+export default Home;
```

Comment:
![OBS-2](https://github.com/jlcarrascof/bookstore-cms/assets/86327867/5ff3084c-caf3-4b8b-bbe2-942ba6321dae)

I deleted the Home component because is not useful in this stage and to respect the clean code rules according to YAGN.